README.md

```diff
@@ -15,43 +15,79 @@ metabolic network, namely Flux Balance Analysis and Flux Variability Analysis.
 [![Tutorial In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/GeomScale/dingo/blob/develop/tutorials/dingo_tutorial.ipynb)
 [![Chat](https://badges.gitter.im/geomscale.png)](https://gitter.im/GeomScale/community?utm_source=share-link&utm_medium=link&utm_campaign=share-link)
 
+
 ## Installation
 
+**Note:** Python version should be 3.8.x. You can check this by running the following command in your terminal:
+```bash
+python --version
+```
+If you have a different version of Python installed, you'll need to install it ([start here](https://linuxize.com/post/how-to-install-python-3-8-on-ubuntu-18-04/)) and update-alternatives ([start here](https://linuxhint.com/update_alternatives_ubuntu/))
+
+**Note:** If you are using `GitHub Codespaces`. Start [here](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/adding-a-dev-container-configuration/setting-up-your-python-project-for-codespaces) to set the python version. Once your Python version is `3.8.x` you can start following the below instructions.
+
+Clone dingo repository
+```bash
+git clone https://github.com/GeomScale/dingo.git
+cd dingo
+```
+
+
+
 To load the submodules that dingo uses, run
 
-````unix
+````bash
 git submodule update --init
 ````
 
 You will need to download and unzip the Boost library:
-```
+```bash
 wget -O boost_1_76_0.tar.bz2 https://boostorg.jfrog.io/artifactory/main/release/1.76.0/source/boost_1_76_0.tar.bz2
 tar xjf boost_1_76_0.tar.bz2
 rm boost_1_76_0.tar.bz2
 ```
 
 You will also need to download and unzip the lpsolve library:
-```
+```bash
 wget https://sourceforge.net/projects/lpsolve/files/lpsolve/5.5.2.11/lp_solve_5.5.2.11_source.tar.gz
 tar xzvf lp_solve_5.5.2.11_source.tar.gz
 rm lp_solve_5.5.2.11_source.tar.gz
 ```
 
 Then, you need to install the dependencies for the PySPQR library; for Debian/Ubuntu Linux, run
 
+```bash
+sudo apt-get update -y
+sudo apt-get install -y libsuitesparse-dev
 ```
-apt-get install libsuitesparse-dev
-```
 
-To install the Python dependencies, install [Poetry](https://python-poetry.org/). Then, run
+To install the Python dependencies, [Poetry](https://python-poetry.org/). 
```

Comment:
Please rephrase. I propose to leave the phrasing as it was before and only change the installation command.

Reply:
Understood.  I've kept the original phrasing as much as possible, except for adding the installation guide for GitHub Codespaces.